lib/personalAccessKey.ts

```diff
@@ -133,6 +133,25 @@ async function getNewAccessToken(
   return accessTokenResponse;
 }
 
+async function getNewAccessTokenByAccountId(
+  accountId: number
+): Promise<AccessToken> {
+  const account = getAccountConfig(accountId) as PersonalAccessKeyAccount;
+  if (!account) {
+    throwErrorWithMessage(`${i18nKey}.errors.accountNotFound`, { accountId });
+  }
+  const { auth, personalAccessKey, env } = account;
+  const authTokenInfo = auth && auth.tokenInfo;
```

Comment:
What happens if `auth.tokenInfo` doesn't exist? Should we abort the call to `getNewAccessToken`?